.github/workflows/dotnet.yml

```
@@ -69,7 +69,7 @@ jobs:
         fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
     - uses: actions/checkout@v2
     - name: Setup .NET
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v2
```

actionlint (1.7.4)

the runner of "actions/setup-dotnet@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)